custom/icds/tasks.py

```diff
@@ -159,3 +165,37 @@ def run_user_indicators(phased_rollout=True):
             if is_monday:
                 run_indicator.delay(domain, user_id, LSSubmissionPerformanceIndicator, language_code)
                 run_indicator.delay(domain, user_id, LSVHNDSurveyIndicator, language_code)
+
+
+@periodic_task(run_every=crontab(minute=0, hour='22'))
+def delete_old_images():
+    if settings.SERVER_ENVIRONMENT not in settings.ICDS_ENVS:
+        return
```

Comment:
Is it possible to _only define_ this periodic task if `settings.SERVER_ENVIRONMENT not in settings.ICDS_ENVS`?

Not a huge deal, but I think that would prevent a no-op task from being enqueued once a day on other environments, which is much more of a clutter/principle issue than a performance one.

I'm thinking either

```
if settings.SERVER_ENVIRONMENT not in settings.ICDS_ENVS:
    @periodic_task(run_every=crontab(minute=0, hour='22'))
    def delete_old_images():
...
```

or

```
def delete_old_images():
...
if settings.SERVER_ENVIRONMENT not in settings.ICDS_ENVS:
    delete_old_images = periodic_task(run_every=crontab(minute=0, hour='22'))(delete_old_images)
```